tests/test_random.py

```diff
@@ -49,3 +50,36 @@ def test_random_check_otput(func):
     for i in range(res.size):
         assert res[i] >= 0.0
         assert res[i] < 1.0
+
+
+def test_randn_normal_distribution():
+    pts = 1000
+    alpha = 0.05
+    dpnp.random.seed(28041990)
+    x = dpnp.random.randn(pts)
+    _, p = stats.normaltest(x)
+    assert p > alpha
+
+
+@pytest.mark.parametrize("func",
+                         [dpnp.random.random,
+                          dpnp.random.random_sample,
+                          dpnp.random.randf,
+                          dpnp.random.sample,
+                          dpnp.random.rand],
+                         ids=['random', 'random_sample',
+                              'randf', 'sample',
+                              'rand'])
+def test_radnom_seed(func):
+    seed = 28041990
+    size = 100
+    shape = (100, 1)
+    if func in [dpnp.random.rand, dpnp.random.randn]:
+        args = size
+    else:
+        args = shape
+    dpnp.random.seed(seed)
+    a1 = func(args)
+    dpnp.random.seed(seed)
+    a2 = func(args)
+    assert_array_equal(a1, a2)

```

Comment:
I'm not sure this function is suitable to compare floating point arrays. I think `assert_allclose` function should be used
Example:
https://github.com/IntelPython/dpnp/blob/b26d624a8841292ed5c976cd59af1fad69ac9c67/tests/test_linalg.py#L59

Reply:
`rtol=1e-05` is suitable for `float` datatype. For `double` it should be much stronger.